assets/config.template.toml

```diff
@@ -17,6 +17,16 @@ gui.lazy_loading = true
 # recommended to set this to "false" as it will lead to the GUI freezing each time you try to zoom in/out.
 dynamic.lazy_loading = true
 
+[ui_settings]
```

Comment:
we can shorten this down to just `ui`

Reply:
already done in my commit btw, I did a lot of refactoring so be ready for merge conflicts if you changed anything :)